base_delivery_carrier_extensions/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2020 FactorLibre - Zahra Velasco <zahra.velasco@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Base delivery carrier extension',
+    'version': '11.0.1.0.0',
+    'depends': [
+        'delivery',
+        'base_delivery_carrier_label'
+    ],
+    'category': 'Stock',
+    'author': 'FactorLibre',
+    'license': 'AGPL-3',
+    'website': 'http://www.factorlibre.com',
+    'data': [
+        'views/stock_view.xml'
+    ],
+    'installable': True,
```

Comment:
'installable' value is True by default in 12.0 version, you can delete it:
https://www.odoo.com/documentation/12.0/reference/module.html

Reply:
This is one of the cases where I prefer to apply the Python zen "Explicit better than implicit", but it's only a matter of taste